Hunting Queries/AzureDevOps/AzDODisplayNameSwapping.yaml

```diff
@@ -0,0 +1,15 @@
+id: cf0c493b-a8af-4b32-8c7e-d4303f3a406f
+name: Azure DevOps Display Name Changes
+description: |
+  'Description:  Shows all users with more than 1 display name in recent history.  This is to hunt for users maliciously changing their display name as a masquerading technique'
+tactics:
+  - Evasion
```

Comment:
Please use the MITRE ATT&CK full names with no spaces. Here Elevation would be PrivilegeEscalation, Evasion would be DefenseEvasion, there is no Social so will need to remove or change to another, also for Hunting queries, we do not require Techniques yet, but we are wanting to start populating them so doing that here would help in the future - https://attack.mitre.org/techniques/enterprise/

Format as follows -

relevantTechniques:

T1078
T1110

Reply:
fixed in next commit